.gitignore

```diff
@@ -18,3 +18,4 @@ nbproject/
 .settings/
 .classpath
 /bin/
+.DS_Store
```

Comment:
Ideally, this would be removed and instead Mac users would use their own global ignore file to ignore these files